static/app/components/charts/baseChart.tsx

```diff
@@ -662,6 +668,95 @@ const ChartContainer = styled('div')<{autoHeightResize: boolean}>`
   }
 `;
 
+// Global styles to inject with the chart. (for portalled tooltips)
+const getPortalledTooltipStyles = (
+  theme: Theme
+): Record<string, Record<string, string | number>> => ({
+  '.chart-tooltip-portal .tooltip-series, .chart-tooltip-portal .tooltip-date': {
```

Comment:
can you link me to the file where the css came from?

Reply:
https://github.com/getsentry/sentry/blob/30663eec9a49d7f1757d41b79d9e29761983c3d1/static/app/components/charts/baseChart.tsx#L570-L669

Reply:
I wonder if something like this would work:

```typescript
import {css} from '@emotion/react';
import styled from '@emotion/styled';

const TooltipStyles = css`... copy+paste it all ...`;

const ChartContainer = styled('div')`
  ${TooltipStyles}
`;

const portalledTooltipStyles = css`
  .chart-tooltip-portal ${TooltipStyles};
`;
```
🤞  the portalled styles do the right thing with nested classnames

Reply:
It seems to work but TS is not happy and it looks like because of the use of `${p => p.theme... }` inside the css string

Is that allowed? I didn't see an examples of that being done in the docs with css
![Screen Shot 2022-09-13 at 2 27 15 PM](https://user-images.githubusercontent.com/3721977/189981782-89b82f8a-5cb1-4ebb-a6c9-0bb4cbf6c97c.png)

Reply:
I just pushed an updated commit I think I got it right based on looking at other uses of css`` in the codebase

Reply:
tbh i don't know any of the ins-and-outs of emotion. I was kind of guessing at what the api might be able to do 😨 

but it's working and i think it's better not to have to copy+paste all those css rules.